frontend/src/components/navbar/burger_drop.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { logout } from '../../actions/session_actions';
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false
+    }
+    this.show = this.show.bind(this);
+    this.close = this.close.bind(this);
+  }
+
+  close() {
+    this.setState(
+      {
+        show: false,
+      },
+      () => {
+        document.removeEventListener("click", this.close);
+      }
+    );
+  }
+
+  show() {
+    this.setState({
+      show: true
+    }, () => {
+      document.addEventListener('click', this.close);
+    })
+  }
+
+  render() {
+    if (this.props.loggedIn)
+    return (
+      <div className="burger-drop">
+        <button
+          className={`hamburger hamburger--emphatic ${
+            this.state.show ? "is-active" : ""
+          }`}
+          type="button"
+          onClick={this.show}
+        >
+          <span className="hamburger-box">
+            <span className="hamburger-inner"></span>
+          </span>
+        </button>
+        {this.state.show ? (
+          <div className="burger-drop-menu gradient">
+            <button
+              className="burger-drop-button logout-button"
+              onClick={this.props.logout}
+            >
+              EJECT<i className="fas fa-sign-out-alt"></i>
+            </button>
+            <button className="burger-drop-button below profile-button">
```

Comment:
I think we will likely want the eject button at the bottom of the list rather than at the top, to mirror convention on most sites. Looks good though!

Reply:
Done!